### PR TITLE
Test method D

### PR DESCRIPTION
**Pull request by JetteSchumann**
Added test for validating the calculation of the classical voronoi data in order to solve #692 

Next steps:
- test of `cut_off` parameter
- test IFD output
- change MA definition after solving #747

### CODE REVIEW

File: systemtest/jpsreport_tests/method_D/D03_corridor_medium_density/D03_corridor_medium_density.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# ---------
+# Test of classical voronoi output
+# Pedestrians are moving in a 6x6 grid in x-direction with v=1 m/s
+# They have a fixed horizontal and vertical distance of 1.0m
+# Due to the alignment the inner voronoi cells should be squares
+# The inner voronoi cells should lie entirely in the specified square measurement area for frame 109
```
Comment (chraibi) [inline systemtest/jpsreport_tests/method_D/D03_corridor_medium_density/D03_corridor_medium_density.py:7]: lay

**Comment by zeroset** [comment]
Can you write the test function in a utility file to avoid copy and pasting the test between low, medium and high resolution tests?

**Comment by JetteSchumann** [comment]
Added tests for cut off option and IFD values.
Next steps:
- add sketches of the scnearios/ testing idea
- implement tests for voronoi cells intersecting with geometry
- still need to change MA definition

**Comment by JetteSchumann** [comment]
### Documentation of the test scenarios

Basic idea is to create a grid of trajectories with pedestrians having a constant vertical and horizontal distance to each other. The voronoi cells of the inner pedestrians (blue area) are supposed to be squares and should not change over time. A measurement area (MA) is constructed at a certain position so that for a specified frame the voronoi cells of the inner pedestrians are entirely included (and only those). The averaged density for the MA should be 1/(area of a square).

Voronoi cells of the outer pedestrians are not validated since they are effected by the geometry or `cut_by_circle`option.

[setup_basic.pdf](https://github.com/JuPedSim/jpscore/files/4550919/setup_basic.pdf)

**Comment by JetteSchumann** [comment]
@gjaeger and @chraibi any further tests needed?
not sure about the `profiles`option...

File: systemtest/jpsreport_tests/CMakeLists.txt [hunk 2]
```diff
@@ -17,7 +26,16 @@ set_tests_properties(
   05_method_A
   01_method_B
   01_method_C
-  01_method_D
+        D01_test_reference_data
```
Comment (zeroset) [inline systemtest/jpsreport_tests/CMakeLists.txt:29]: please use the same indention

Comment (JetteSchumann) [reply systemtest/jpsreport_tests/CMakeLists.txt]: Done. Happened when automatically reformating the code...

**Pull request merged**